conans/client/generators/cmake_find_package_multi.py

```diff
@@ -299,7 +313,7 @@ def content(self):
                 find_lib = target_template.format(name=pkg_findname, deps=deps,
                                                   build_type_suffix=build_type_suffix,
                                                   deps_names=deps_names)
-                ret["{}Target-{}.cmake".format(pkg_filename, build_type.lower())] = find_lib
+                ret["{}Target-{}.cmake".format(pkg_filename, self.configuration.lower())] = find_lib
```

Comment:
so, configuration no longer matches build type?

Reply:
It matches by default the ``build_type``. But the user can have custom configurations, like ``ReleaseShared``, so this allows the user to provide and manage those cases too.